layers/+window-management/exwm/config.el

```diff
@@ -32,6 +32,15 @@
 (defvar exwm-leader-key nil
   "Key to use for EXWM global commands")
 
+(defvar exwm-enable-systray nil
+  "Whether to enable EXWM's bundled system tray implementation.")
+
+(defvar exwm-autostart-xdg-applications nil
+  "Whether to run $XDG_USER_HOME/autostart applications after initialization.")
+
+(defvar exwm-custom-init nil
+  "This can be set to a function that runs after all other EXWM initialization.")
+
 (defvar exwm-workspace-switch-wrap t
   "Whether `exwm/exwm-workspace-next' and `exwm/exwm-workspace-prev' should wrap.")
```

Comment:
Where is this variable used?

Reply:
Line 218 of packages.el.

Reply:
No I cannot find it
https://github.com/be11ng/spacemacs/blob/6e0fbaefde4c1e3424fc36e2572d7e77e581b0e4/layers/%2Bwindow-management/exwm/packages.el#L218

Reply:
Sorry – line 219.
https://github.com/be11ng/spacemacs/blob/6e0fbaefde4c1e3424fc36e2572d7e77e581b0e4/layers/%2Bwindow-management/exwm/packages.el#L219